roles/ovirt-host-deploy-firewalld/vars/firewalld_rules_common.yml

```diff
@@ -0,0 +1,9 @@
+host_deploy_firewalld_rules:
+  ## Service definition:
+  - service: rpc-bind
+  - service: snmp
+  - service: cockpit
+
+  ## Port definition:
+  # ovirt-imageio-daemon
+  - port: 54322/tcp
```

Comment:
So no need, as it's part of the firewalld 0.4.4.5 release:

 https://github.com/firewalld/firewalld/blob/master/config/services/ovirt-imageio.xml